pandas/core/generic.py

```diff
@@ -808,11 +813,11 @@ def swaplevel(self, i=-2, j=-1, axis=0):
         We *highly* recommend using keyword arguments to clarify your
         intent.
 
-        >>> df.rename(index=str, columns={"A": "a", "B": "c"})
-           a  c
-        0  1  4
-        1  2  5
-        2  3  6
+        >>> df.rename(index="index_{}".format, columns={"A": "a", "B": "c"})
```

Comment:
any use of ``.add_prefix/suffix`` in the main docs? if so can you update

Reply:
No, it's used in the docs. It's shown in ``10min_to_pandas`` as a result of tab completion, but that should stay until removal.

Reply:
no please change the docs now

Reply:
Sorry, I miswrote: it's **not** in the docs, expect as a result of that tab comletion example.

Reply:
ahh these need to be added to
_deprecations in series and dataframe

Reply:
Ok, added.